addons/mail/static/src/models/thread/thread.js

```diff
@@ -1300,51 +1291,48 @@ function factory(dependencies) {
          */
         _computeMessageAfterNewMessageSeparator() {
             if (this.model !== 'mail.channel') {
-                return [['unlink']];
+                return unlink();
             }
             if (this.localMessageUnreadCounter === 0) {
-                return [['unlink']];
+                return unlink();
             }
             const index = this.orderedMessages.findIndex(message =>
                 message.id === this.lastSeenByCurrentPartnerMessageId
             );
             if (index === -1) {
-                return [['unlink']];
+                return unlink();
             }
             const message = this.orderedMessages[index + 1];
             if (!message) {
-                return [['unlink']];
+                return unlink();
             }
-            return [['link', message]];
+            return link(message);
         }
 
         /**
          * @private
          * @returns {mail.message[]}
          */
         _computeOrderedMessages() {
-            return [['replace', this.messages.sort((m1, m2) => m1.id < m2.id ? -1 : 1)]];
+            return replace(this.messages.sort((m1, m2) => m1.id < m2.id ? -1 : 1));
         }
 
         /**
          * @private
          * @returns {mail.message[]}
          */
         _computeOrderedNonTransientMessages() {
-            return [['replace', this.orderedMessages.filter(m => !m.isTransient)]];
+            return replace(this.orderedMessages.filter(m => !m.isTransient));
         }
 
         /**
          * @private
          * @returns {mail.partner[]}
          */
         _computeOrderedOtherTypingMembers() {
-            return [[
-                'replace',
-                this.orderedTypingMembers.filter(
-                    member => member !== this.env.messaging.currentPartner
-                ),
-            ]];
+            return replace(this.orderedTypingMembers.filter(
+                member => member !== this.env.messaging.currentPartner
+            ));
```

Comment:
Isn't this just equivalent to doing `unlink(this.env.messaging.currentPartner)`?

Reply:
The method is to compute `orderedOtherTypingMembers` via `orderedTypingMembers`. I do not think it can be replaced by `unlink()`

Reply:
Oh you are correct, I thought it was just to remove the current partner from `orderedTypingMembers` but it is to store it in another field.